k8s/namespaces/jenkins/patches/ptl/cluster-00/jenkins.yaml

```
@@ -45,13 +45,13 @@ spec:
                       - key: AKS_AAT_SUBSCRIPTION_NAME
                         value: DTS-SHAREDSERVICES-TEST
                       - key : AAT_AKS_KEY_VAULT
-                        value: dtssharedserviceskv
+                        value: dtssharedservicestestkv
                       - key: AKS_PROD_SUBSCRIPTION_NAME
                         value: DTS-SHAREDSERVICES-PROD
                       - key: PIPELINE_METRICS_URL
                         value: https://sds-jenkins-pipeline-metrics.documents.azure.com/
                       - key: TF_STATE_STORAGE_TEMPLATE
-                        value: sdsstate
+                        value: sdsstateprod
```

this template was correct
```suggestion
                        value: sdsstate
```